config/settings.d/dns_nsupdate.yml.example

```diff
@@ -0,0 +1,16 @@
+---
+#
+# Configuration file for 'nsupdate' dns provider
+#
+# Can be true, false
+:enabled: true
+
+#:dns_key: /etc/rndc.key
+# use this setting if you are managing a dns server which is not localhost though this proxy
+#:dns_server: dns.domain.com
+# use this setting if you want to override default TTL setting (86400)
+#:dns_ttl: 86400
```

Comment:
To wrap this up:

* get rid of the dns_provider
* move dns_ttl to the dns.yaml file